### DISCUSSION TITLE
Responsive Issues

### DISCUSSION THREAD

**Discussion by kemmieg** [twbs/bootstrap#39632] [Q&A]
I'm having an issue with responsiveness. I am pulling all the css for bootstrap using the cdn and when I view it on chrome with the iPhone12 setting replicates what I see on my iPhone 14. Otherwise when I look at it on anything other than the device it looks correct. I have a custom css file and I tried removing all my css and just viewing it by itself and it's still not responsive. I did some searching and all I could find was to add this snippet `<meta name="viewport" content="width=device-width, initial-scale=1">` which is already added. I feel like I've lost my mind not being able to figure out why my site is not responsive on phones. Can someone help me out? I have it as a sub domain on my server so it can be viewed the url is [https://dbkclient.com/tests/index.php?page=home](https://dbkclient.com/tests/index.php?page=home). Any assistance would be greatly appreciated.

**Comment by gerardnico** [comment]
I can see one issue.

When you use a row, to have a mobile friendly template, you need to set it on 12 for the col (ie takes the whole width)

Example:
Not
```html
<div class="col-4 text-center">
   <img src="assets/images/example_class_placeholder.webp">
</div>
```

But
```html
<div class="col-12 col-md-4 text-center">
   ....
</div>
```

**Reply by kemmieg** [reply]
That makes sense. I will go through my code and make those changes. Thank you

**Reply by kemmieg** [reply]
I went through the first page and added the col-12 and that had no effect on the layout. Also, below you mentioned using f12 for something. I. have never heard of what you are talking about can you please explain what you mean by the instructions below?
Thank you

**Reply by gerardnico** [reply]
https://developer.chrome.com/docs/devtools/dom/

**Comment by gerardnico** [comment]
If you get stuck, just type F12 and just delete the node in the elements pan.
This way, you can see the element that is perturbing your mobile friendly design.
